src/btree/bt_read.c

```diff
@@ -311,7 +311,7 @@ __evict_force_check(WT_SESSION_IMPL *session, WT_PAGE *page)
 	__wt_txn_update_oldest(session, 0);
 
 	/* If eviction cannot succeed, don't try. */
-	return (__wt_page_can_evict(session, page, 1, NULL));
+	return (__wt_page_can_evict(session, page, 1));
```

Comment:
I'm missing something about this change -- the common case of forced eviction is appends, and in that case we want to do an in-memory split before trying ordinary eviction.  Won't this change prevent some of those pages being split, because we're no longer checking if they can be?